### PR TITLE
Add eslint and fix linting errors.

### PR DESCRIPTION
**Pull request by nigelhanlon**
#### This resolves Clubhouse issue: Add eslint to flyway-api-demo

- [x] Tests have been written
- [x] All TODOs have a task created and assigned
- [x] Any tech debt raised as cards in Clubhouse
- [x] Checked that the new code runs properly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,12 +25,16 @@
     "fastify": "^0.41.0",
     "fastify-plugin": "^0.2.2",
     "lodash": "^4.17.4",
-    "original-url": "^1.2.1"
+    "original-url": "^1.2.1",
+    "@condenast/copilot-js-sdk": "^10.0.0",
+    "@condenast/flyway-schema-validator": "^0.6.1"
```
Comment (srilq) [inline package.json]: Why was this moved out of devDeps?

File: src/controllers/getArticle.spec.js [hunk 2]
```diff
@@ -35,6 +36,7 @@ describe('GET /api/articles/:id', () => {
       expect(body.brand).toBe(DEFAULT_BRAND);
       await expect(validate(body)).resolves.toMatchObject(body);
     }
+    /* eslint-enable no-restricted-syntax, no-await-in-loop */
```
Comment (srilq) [inline src/controllers/getArticle.spec.js]: Why not disable this in `eslintrc`?

Comment (nigelhanlon) [reply src/controllers/getArticle.spec.js]: I don't mind disabling for a few lines in a specific test, but adding to `.eslintrc` would disable the rules globally.
